language/stdlib/transaction_scripts/add_to_script_allow_list.move

```diff
@@ -2,6 +2,28 @@ script {
 use 0x1::LibraTransactionPublishingOption;
 use 0x1::SlidingNonce;
 
+/// # Sender Preconditions
+/// The sending account (`lr_account`) must be the Libra Root account. The script allow
+/// list must not already hold the script `hash` being added.
+/// # Accounts Affected
+/// The Libra Root account is affected with normal transaction effects (sequence number,
+/// nonce). Additionally the on-chan config
+/// `LibraTransactionPublishingOption::LibraTransactionPublishingOption`'s
+///`script_allow_list` field is updated to contain the new script `hash`.
+/// # Accounts Created
+/// None
+/// # Description
+/// Adds the script hash `hash` to the script allow list. After this transaction is
+/// committed, transactions with this `hash` may be sent to the network.
```

Comment:
I wonder if we should say "script executes successfully" or choose some other terminology instead of "transaction is committed". "is committed" seems both too vague (e.g., it might have committed and aborted) and too specific (e.g., two dependent transactions can be in the same block rather than waiting for the first to be committed).

I'm ok with any terminology, but think that we should be careful/consistent here...

Reply:
Agreed on terminology. I think this is another thing to bring up tomorrow morning as well possibly. 

I'm leaning towards "is executed successfully and committed" since that should be unambiguous. And I think there's a distinction to be made (that may be too technical, so fine not making it) between "executed successfully" and "executed successfully and committed" especially around reconfigurations. In my thinking the former is saying "dependencies within the block are ok" and the latter is saying "dependencies on this within the block aren't"?